source/RevitLookup/Core/ComponentModel/Descriptors/ViewScheduleDescriptor.cs

```diff
@@ -0,0 +1,199 @@
+            nameof(ViewSchedule.GetDefaultNameForSheetList) => ResolveSet.Append(ViewSchedule.GetDefaultNameForSheetList(context)),
+            nameof(ViewSchedule.GetDefaultNameForViewList) => ResolveSet.Append(ViewSchedule.GetDefaultNameForViewList(context)),
+            nameof(ViewSchedule.GetValidFamiliesForNoteBlock) => ResolveSet.Append(ViewSchedule.GetValidFamiliesForNoteBlock(context)),
+#if REVIT2022_OR_GREATER
+            nameof(ViewSchedule.GetScheduleInstances) => ResolveScheduleInstances(),
+            nameof(ViewSchedule.GetSegmentHeight) => ResolveSegmentHeight(),
+#endif
+            _ => null
+        };
+        
+        ResolveSet ResolveStripedRowsColor()
+        {
+            var values = Enum.GetValues(typeof(StripedRowPattern));
+            var resolveSummary = new ResolveSet(values.Length);
+            
+            foreach (StripedRowPattern pattern in values)
+            {
+                resolveSummary.AppendVariant(viewSchedule.GetStripedRowsColor(pattern), pattern.ToString());
+            }
+            
+            return resolveSummary;
+        }
+        
+        ResolveSet ResolveValidTextTypeId()
+        {
+            var types = context.EnumerateTypes<TextNoteType>().ToArray();
+            var resolveSummary = new ResolveSet(types.Length);
+            
+            foreach (var type in types)
+            {
+                var result = viewSchedule.IsValidTextTypeId(type.Id);
+                resolveSummary.AppendVariant(result, $"{type.Name}: {result}");
+            }
+            
+            return resolveSummary;
+        }
+        
+        ResolveSet ResolveDefaultNameForKeySchedule()
+        {
+            var categories = ViewSchedule.GetValidCategoriesForKeySchedule();
+            var resolveSummary = new ResolveSet(categories.Count);
+            foreach (var id in categories)
+            {
+                resolveSummary.AppendVariant(ViewSchedule.GetDefaultNameForKeySchedule(context, id));
+            }
+            
+            return resolveSummary;
+        }
+        
+        ResolveSet ResolveDefaultNameForMaterialTakeoff()
+        {
+            var categories = ViewSchedule.GetValidCategoriesForMaterialTakeoff();
+            var resolveSummary = new ResolveSet(categories.Count);
+            foreach (var id in categories)
+            {
+                resolveSummary.AppendVariant(ViewSchedule.GetDefaultNameForMaterialTakeoff(context, id));
+            }
+            
+            return resolveSummary;
+        }
+        
+        ResolveSet ResolveDefaultNameForSchedule()
+        {
+            var categories = ViewSchedule.GetValidCategoriesForSchedule();
+            var resolveSummary = new ResolveSet(categories.Count);
+            foreach (var id in categories)
+            {
+                if (id == new ElementId(BuiltInCategory.OST_Areas)) continue;
```

Comment:
What is wrong with OST_Areas ? And you can move new ElementId() before foreach, for optimization

Reply:
OST_Areas needs to have AreaSchemeId, otherwise throws an exception. I added AreaSchemeId in such case and move new ElementId outside a loop